analyticConfigurations/computed-aws.nlb.json

```diff
@@ -0,0 +1,19 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "netuitive.aws.networkelb.unhealthyhostpercent",
+        "properties": {
+          "expression": "(data['aws.networkelb.unhealthyhostcount'].actual + data['aws.networkelb.healthyhostcount'].actual) == 0 ? 0 : (data['aws.networkelb.unhealthyhostcount'].actual / (data['aws.networkelb.unhealthyhostcount'].actual + data['aws.networkelb.healthyhostcount'].actual))*100",
```

Comment:
If I'm understanding this correctly the value will be 0% if there are no hosts at all?